__tests__/app.test.js

```diff
@@ -112,3 +112,66 @@ describe('/api/articles', () => {
     });
   });
 });
+
+describe('/api/articles/:article_id/comments', () => {
+  it('responds with status 200 and all comments for an existing article with the correct properties', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSortedBy('created_at', { descending: true });
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+
+  it('responds with status 200 and all comments for an existing article with the correct properties', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSortedBy('created_at', { descending: true });
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+
+  it('responds with status 200 and an empty array when there are no comments for a valid article', () => {
+    return request(app)
+      .get('/api/articles/2/comments')
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(0);
+      });
+  });
+
+  it('responds with status 404 and a message for a non-existent article', () => {
+    return request(app)
+      .get('/api/articles/999999/comments')
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe('Article not found');
+      });
+  });
```

Comment:
Nice error handling, but dont forget to test for when an invalid article id is given. I.e. /api/articles/banana/comments